lisa/sut_orchestrator/azure/common.py

```diff
@@ -1448,7 +1448,9 @@ def load_environment(
         assert isinstance(node, RemoteNode)
 
         vm = vms_map.get(node.name)
-        assert_that(vm).is_not_none()
+        assert_that(vm).described_as(
+            f"VM details for {node.name} is {vm}. This is an unexpected failure"
```

Comment:
This message just prints values, no reason or solution. It should be like, "Cannot find vm with name {node.name}. Make sure the VM exists in resource group {resource_group_name}"

Reply:
@squirrelsc I initially thought of something like this, but 
vms = compute_client.virtual_machines.list(resource_group_name) 
this api dynamically fetches the vm from the resource group right?

Reply:
Yes, so if the vm is not found. The error message indicates the expected vm in a resource group.

Reply:
Okay, Updated